run-tests.py

```diff
@@ -315,7 +315,13 @@ def export_notebook(ipath, opath):
         description="Run unit tests for PyBaMM.",
         epilog="To run individual unit tests, use e.g. '$ tests/unit/test_timer.py'",
     )
+
     # Unit tests
+    parser.add_argument(
+        "--integration",
+        action="store_true",
+        help="Which folder to run the tests from.",
```

Comment:
Can you update help to say "Run integration tests using the `python` interpreter."

Reply:
We can do both either` python run-tests.py --unit --folder all` or `python run-tests.py --folder all`. It will work fine for both cases.